big_tests/Makefile

```
@@ -4,6 +4,7 @@ all: test
 # user overridable
 TESTSPEC ?= default.spec
 PRESET   ?= all
+CT_HOOKS ?= ""
```

This one isn't needed. If we prefer to keep it to remind us about the existence of the variable, then I would change it to the following:

```suggestion
CT_HOOKS ?=
```

---

The intention is to have a list of user-overridable variables, see the comment in line 4.
If you think that `""` looks verbose, I can remove it, it makes no big difference to be honest.